bindata/network/ovn-kubernetes/common/003-rbac-controller.yaml

```diff
@@ -20,7 +20,6 @@ rules:
   - watch
 - apiGroups: [""]
   resources:
-  - namespaces/status
```

Comment:
closing PR: Pat told me we don't do any namespaces/status updates starting https://github.com/openshift/cluster-network-operator/commit/962798c9b76addb8e06c04a7475a9b53415a66b8#diff-cd68b4caec2e8a6afb4c120d980469834c62524b90a300f2adf90b6d6bec0c55L75-L76